src/dex/idle-dao/tokens.ts

```diff
@@ -0,0 +1,90 @@
+import { Token } from '../../types';
+import { Network } from '../../constants';
+
+import { IdleToken } from './types';
+import { IDexHelper } from '../../dex-helper';
+import CDO_ABI from '../../abi/idle-dao/idle-cdo.json';
+import { AbiItem } from 'web3-utils';
+
+const poolsByTokenAddress: { [address: string]: IdleToken[] } = {};
```

Comment:
Should not it be also split by chain? like `const poolsByTokenAddress: {[network: number]: {[address: string]: IdleToken[]} } = {};`

Reply:
done